platform/android/java/lib/src/org/godotengine/godot/input/GodotGestureHandler.kt

```diff
@@ -197,8 +197,13 @@ internal class GodotGestureHandler : SimpleOnGestureListener(), OnScaleGestureLi
 		if (event.actionMasked == MotionEvent.ACTION_UP) {
 			nextDownIsDoubleTap = false
 			GodotInputHandler.handleMotionEvent(event)
+			return true
 		}
-		return true
```

Comment:
I am not sure why this was always returning true even for cases when it wasn't actually handling the event. I've changed it to only return true when it handles the event, consistent with the other event listeners here.

Reply:
This doesn't look correct, this callback should receive both down and up events and with this update the logic now returns false for down events, which would have the effect of cancelling the next up event.
Can you log the events passing through this method and check we are still receiving the expected flow.